volumes.c

```diff
@@ -155,6 +174,34 @@ static int pv_volumes_mount_volume(struct pantavisor *pv, struct pv_volume *v)
 			pv_log(INFO, "mounting proper .data dir");
 			mkdir_p(mntpoint, 0755);
 			ret = mount(path, mntpoint, NULL, MS_BIND | MS_REC, NULL);
+		} else if (handler) {
```

Comment:
I guess this would be nicer in a separate function like @parthitce does in https://github.com/pantavisor/pantavisor/pull/16

Reply:
bu isnt that cleanup something his PR brings?